editor/editor_inspector.cpp

```diff
@@ -579,6 +579,13 @@ void EditorProperty::_notification(int p_what) {
 			}
 		} break;
 		case NOTIFICATION_ENTER_TREE: {
+			// Ensure shortcuts on properties are only triggered in the context of the inspector they are contained within.
+			EditorInspector *inspector = get_parent_inspector();
+			if (inspector) {
+				inspector->get_root_inspector();
```

Comment:
This does nothing.

Reply:
Ahaha whoops, at the time I didn't realise that getting the root was possible so I thought maybe I needed to loop through parents to get the root. Pretty funny mistake though lol

Reply:
Hmm never mind, this did have intention. `get_parent_inspector()` can return `nullptr`. Therefore we need to protect against it returning returning `nullptr` before calling `get_root_inspector()`. There is no way to get the root inspector directly from an `EditorProperty`

Reply:
updated to `inspector = inspector->get_root_inspector();`